src/main/cpp/mechanisms/ClimberManager/ManualClimbState.cpp

```diff
@@ -76,6 +81,8 @@ bool ManualClimbState::AtTarget()
 bool ManualClimbState::IsTransitionCondition ( bool considerGamepadTransitions )
 {
 	// To get the current state use m_mechanism->GetCurrentState()
-	return false;
+	if (m_mechanism->IsClimbMode()==true){
```

Comment:
Just return m_mechanism->IsClimbMode, because you don't have a return false with this current setup